lib/truffle/truffle/patching.rb

```diff
@@ -13,11 +17,16 @@ module Kernel
   alias_method :require_without_truffle_patching, :require
 
   def require(path)
```

Comment:
I think it's a shame to have to pile on another monkey patch of `require`. Could we do anything else instead?